karton/system/system.py

```diff
@@ -31,6 +31,7 @@ class SystemService(KartonServiceBase):
     TASK_DISPATCHED_TIMEOUT = 24 * 3600
     TASK_STARTED_TIMEOUT = 24 * 3600
     TASK_CRASHED_TIMEOUT = 3 * 24 * 3600
+    TASK_TRACKING_TTL = 60 * 60 * 24 * 30
```

Comment:
Probably it would be better to use unified hours (`3600` instead of `60 * 60`): `30 * 24 * 3600`

Reply:
sure, I can do it. I've just copy&pasted from the original place :)